src/main/java/com/mpatric/mp3agic/ID3Wrapper.java

```diff
@@ -114,10 +114,10 @@ public void setYear(String year) {
 	}
 
 	public int getGenre() {
-		if (id3v1Tag != null && id3v1Tag.getGenre() != -1) {
-			return id3v1Tag.getGenre();
-		} else if (id3v2Tag != null) {
+		if (id3v2Tag != null && id3v2Tag.getGenre() != -1) {

```

Comment:
This changes the default behaviour and would lead to different results when a consumer updates mp3agic.
We could do that in the future but should keep the decision to prefer id3v2 over id3v1 consistent in the whole library. This should go to another pull request and signed off by @mpatric before.